### DISCUSSION TITLE
v1.20.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#3023] [Releases & Announcements]
We have released a new version of Chatwoot v1.20.0 with the following updates.

### 🔥 New Channels: Line & Telegram

We have added native support for Line and Telegram in v1.20.0. As of now, text messages are supported, we will be adding support for images/files in the upcoming releases.

![Group 2(1)](https://user-images.githubusercontent.com/2246121/133566938-3ce32346-1165-49b3-beeb-083a437ac566.png)

### 🤡 CSAT surveys for every channel

You can now opt to send a Customer satisfaction surveys on every channel once the conversation is resolved. The user would receive a link where they can register their feedback.

A example on how it works on twitter is shown below.

![Bitmap](https://user-images.githubusercontent.com/2246121/133566959-e9ec4e8b-10dd-40b9-8aa0-50ac8f312106.png)
<img width="701" alt="Screenshot-2021-09-16-at-12 17 07-PM" src="https://user-images.githubusercontent.com/2246121/133567017-e5ecd6d8-3b00-4eca-aa73-273a82225398.png">

### 💬 Send MMS via Twilio

Chatwoot now supports sending Multimedia message via Twilio API.

### 🎨 Updated mention notification emails

Earlier the mention notification emails didn't have the message content. We have updated the design to include the message content as well.

<img width="589" alt="Screenshot-2021-09-15-at-8 15 09-PM" src="https://user-images.githubusercontent.com/2246121/133567070-48ffb077-b29b-4d9d-a0fd-3201267c1a5e.png">

### Bug fixes & Improvements:

- For emails forwarded from Google Groups, Google rewrites the FROM address to the group email and the original email will be available under X-Original-Sender. The new update would handle this case.
- Added the APIs and UI for custom attributes schema. (upcoming feature)
- Added the ability to remove Inbox avatar.
- Update the design of the web-widget for WCAG 2.0 compliance. Thanks to @koudshoorn for the PR.
- Added multiline support for the greeting messages.
- Support more special characters in the password.
- You can now pop out the reply box to write longer text messages.

Thanks to @wilso199, @koudshoorn, @SanjuSudheerM, @noobnooc for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v1.20.0'>v1.20.0</a>.</em>

**Comment by helpobras** [comment]
I'm using version 1.19 self hosted on ubuntu. can I follow this upgrade upgrade tutorial exactly as described? https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm

**Reply by sojan-official** [reply]
yes you can.
